app/src/main/java/org/thunderdog/challegram/component/sticker/StickerSetWrap.java

```diff
@@ -337,7 +374,7 @@ public void onFactorChangeFinished (int id, float finalFactor, FactorAnimator ca
 
   private void setInProgress (boolean inProgress) {
     if (this.inProgress != inProgress) {
-      this.inProgress = inProgress;
+      UI.post(() -> this.inProgress = inProgress);
```

Comment:
Why exactly this change was needed? This method must be always called from the main thread, as otherwise view-related methods will cause a crash with `CalledFromWrongThreadException`.